app/src/main/java/com/geeksville/mesh/ui/LinkedCoordinates.kt

```diff
@@ -40,7 +40,9 @@ fun LinkedCoordinates(
         val annotatedString = buildAnnotatedString {
             pushStringAnnotation(
                 tag = "gps",
-                annotation = "geo:${position.latitude},${position.longitude}?z=17&label=${
+                // URI scheme is defined at:
+                //  https://developer.android.com/guide/components/intents-common#Maps
+                annotation = "geo:0,0?q=${position.latitude},${position.longitude}&z=17&label=${
                     URLEncoder.encode(name, "utf-8")
                 }"
```

Comment:
While this change doesn't affect it, passing in the label as a query param for the URI doesn't seem to be supported for Google maps (or other providers for that matter). We _could_ do the Maps-intent-specific format of placing the location label in parentheses after the lat/long, but that seems to break things like OsmAnd as it just attempts to include the label as a part of the geo search.

Not sure who makes the determination on what is the best UX/solution, but something to think about for future improvements.